vita3k/modules/SceLibKernel/SceLibKernel.cpp

```diff
@@ -1101,8 +1101,15 @@ EXPORT(int, sceKernelGetThreadEventInfo) {
     return UNIMPLEMENTED();
 }
 
-EXPORT(int, sceKernelGetThreadExitStatus) {
-    return UNIMPLEMENTED();
+EXPORT(int, sceKernelGetThreadExitStatus, SceUID thid) {
+    const ThreadStatePtr thread = lock_and_find(thid ? thid : thread_id, host.kernel.threads, host.kernel.mutex);
```

Comment:
Does this condition `thid ? thid : thread_id` really exists?
I was expecting the following to be true : `sceKernelGetThreadExitStatus(NULL) == SCE_KERNEL_ERROR_UNKNOWN_THREAD_ID`.
Or is it a copy-paste from our `sceKernelChangeThreadPriority` or `sceKernelGetThreadInfo` implementation?

Reply:
It's a copy-paste.

Reply:
Ok, it should be fine as it is. It can be improved afterwards.

Reply:
Null or 0 means current thread